deployment/terraform/live/main.tf

```diff
@@ -24,3 +24,14 @@ module "identity_pool" {
   upload_bucket_name       = "${module.object_storage.upload_bucket_name}"
   iam_resource_name_prefix = "${terraform.workspace}"
 }
+
+module "docker_host" {
+  source = "../modules/docker_host"
+  cidr_block = "${var.cidr_block}"
+  vpc_id = "${module.vpc.vpc_id}"
+}
+
+module "vpc" {
+  source = "../modules/vpc"
+  cidr_block = "${var.cidr_block}"
```

Comment:
add something like: `security_group_name = "${terraform.workspace}-docker"` to avoid multiple SGs in the account with the same (default) name